lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderAccessService.java

```diff
@@ -424,6 +464,10 @@ public void handleAccess(HttpServletRequest req, HttpServletResponse res, Refere
 					SimplePage currentPage = simplePageToolDao.getPage(item.getPageId());
 					String owner = currentPage.getOwner();  // if student content
 					String group = currentPage.getGroup();  // if student content
+					
+					//If owner != null it's a student page (Does group factor into this?)
```

Comment:
Do we have an answer on if groups make a difference here? We do have sites that make heavy use of them.

Reply:
Yeah, I didn't test this so I'm not 100% sure. It seems like it identifies if the page is a student page by having the page owner != null, but I guess it can also be a "group student page".

It's quite possible this check should be
boolean studentcontent = (owner != null || group != null);

Let me double check this too.

Reply:
Yeah, it seems like they do, but I'm also seeing that all group student pages also have an owner as the first student that creates it. But I don't know if this is necessarily true. I am not seeing any cases of this query returning anything. So I don't think it really matters, but having an || case in here makes sense to me.

SELECT * FROM sakai_ilstu_prod.lesson_builder_pages where groupid is not null and owner is null;